src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/BegrunnelserForPeriodeContext.kt

```diff
@@ -374,12 +398,28 @@ class BegrunnelserForPeriodeContext(
             .mapNotNull { (aktør, tidslinje) ->
                 val person =
                     personopplysningGrunnlag.personer.find { it.aktør.aktivFødselsnummer() == aktør.aktivFødselsnummer() }
+                        ?: return@mapNotNull null
+
+                // VilkårResultat opphører samtidig som et annet blir oppfylt etter forskyvning
```

Comment:
Dette blir litt 🤯 for meg. 

Om `tidslinje.erTom() == true` er det samme som av VilkårResultat opphører samtidig som et annet blir oppfylt etter forskyvning tror jeg vi bør endre på navn av variablene. 

Jeg tolker det sånn at om førskjøvetOppfylteVilkårResultatTidslinje er tom så har aldri personen noen oppfylte vilkår, men det stemmer kanskje ikke?

Reply:
Det stemmer at vi ikke har noen oppfylte vilkår i det tilfelle, men vi ønsker allikevel å ha med vilkårresultatene som kommer før "skiftet" for å kunne begrunne de.